### ISSUE TITLE
can you remove auto-complete dependency?

### ISSUE THREAD

**Issue by stardiviner** [kiwanami/emacs-edbi#33]
Some people (like me) don't use auto-complete, we use `company-mode` instead.
I hope you can refactor edbi to use `completion-at-point-functions` (which is used by Emacs default, and supported by company-mode). it is better solution.

**Comment by kiwanami** [comment]
Hi @stardiviner 
Yes. your advice is reasonable. I would do that.

**Comment by stardiviner** [comment]
great! thanks. Very powerful extension.

**Comment by stardiviner** [comment]
I think the best way is to use `completion-at-point-functions`.

**Comment by ReneFroger** [comment]
Any update on this? I think a majority of Emacs users use `company-mode` nowadays.

**Comment by stardiviner** [comment]
@ReneFroger Yes, but `company-mode` support `company-capf` backend which uses `completion-at-point-functions`.

**Comment by kiwanami** [comment]
I have implemented simple capf function. I confirmed `company-mode` completes the words.

But, because I'm not sure the specification of `completion-at-point-function`, I could not display some useful information beside the completion items, such as item types, column data-types.

Is it help company users?

**Comment by stardiviner** [comment]
I forgot to update this issue, I found a new extension [company-edbi](https://github.com/proofit404/company-edbi) which works great. Maybe can merge together?

**Comment by Valdaer** [comment]
And maybe release a 0.1.3 version of edbi so both emacs-edbi and company-edbi would be installable from melpa stable? https://github.com/kiwanami/emacs-edbi/issues/38
